Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -547,10 +548,9 @@ class ITinyMCEPluginSchema(Interface):
             SimpleTerm('visualchars', 'visualchars', u"visualchars"),
             SimpleTerm('wordcount', 'wordcount', u"wordcount")
         ])),
-        default=['advlist', 'directionality', 'emoticons',
-                 'fullscreen', 'hr', 'insertdatetime', 'lists', 'media',
+        default=['advlist', 'fullscreen', 'hr', 'lists', 'media',
                  'nonbreaking', 'noneditable', 'pagebreak', 'paste', 'preview',
-                 'print', 'save', 'searchreplace', 'tabfocus', 'table',
+                 'print', 'save', 'tabfocus', 'table',
```

Comment:
I know you removed the ``save`` option on purpose from the vocabulary. but you did not remove it from the ``default`` list. instead you removed ``searchreplace``.